gearmand/setup.py

```diff
@@ -27,7 +27,7 @@ def get_dependencies():
         return f.readlines()
 
 
-CHECKS_BASE_REQ = 'datadog-checks-base'
+CHECKS_BASE_REQ = 'datadog-checks-base>=11.0.0'
```

Comment:
not sure if too important, but the latest version seems to be 13.1.0 now: https://pypi.org/project/datadog-checks-base/#history

Reply:
I believe 11.0.0 is enough, so let's keep the lowest version https://github.com/DataDog/integrations-core/pull/5903/files#r392976513